.yamato/csharp-tests.yml

```diff
@@ -12,12 +12,16 @@ test_mac_editmode_{{ editor.version }}:
   variables:
     UNITY_VERSION: {{ editor.version }}
   commands:
-    - ./run-tests-editmode-osx-editor.sh
+    - python -u ml-agents/tests/yamato/editmode_tests.py
```

Comment:
`-u` means "unbuffered output", otherwise we get all the python output after the unity output.